docs/account-experience/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+id: index
+title: Account-Experience Overview
+sidebar_label: Account-Experience
+---
+
+The Ory Account Experience is the default user interface for all self-service screens like login, registration, or consent. It can
+be accessed under `https://your-slug.projects.oryapis.com/ui`. New Ory Network projects are automatically configured to use the
+Account Experience. You can find various customizations and settings in the Ory Console under `Account Experience`.
+
+## Theming
+
+The Account Experience can be themed using the Ory Console. You can find the theme settings under `Account Experience` >
```

Comment:
Use a direct link :) This makes it so much easier to find if the descriptions ever change in the UI!

Here for example: https://console.ory.sh/projects/current/account-experience/theming